src/plist.h

```diff
@@ -238,8 +238,29 @@ extern void            AssPlist (
 
 /****************************************************************************
 **
+*F  ADD_PLIST( <list>, <val> ) . . . . . . . .  assign to end of a plain list
+**
+**  Note that this function does not adjust the TNUM of the list object. It
+**  also does not attempt to convert the list to a different representation,
+**  such as a string or blist. If your need that, use AddList or AddPlist
+**  instead.
+**
+*/
+static inline UInt ADD_PLIST(Obj list, Obj val)
+{
+    const UInt pos = LEN_PLIST(list) + 1;
+    GROW_PLIST( list, pos );
+    SET_LEN_PLIST( list, pos );
+    SET_ELM_PLIST( list, pos, val );
+    if (IS_BAG_REF(val))
```

Comment:
Well, the reason I put this in here is that some code that I changed to use `ADD_PLIST` was clever enough to not invoked `CHANGED_BAG` when inserting intobjs or NULL pointers. So I figured this was safe optimization.

I am not sure what the helper function would gain, other than being yet another function you'd have to know by heart in order to read kernel code ;-). Perhaps if it did something more elaborate than the two lines it'd replace here?